packages/core/src/Provider/NetworkInfoProvider/types.ts

```diff
@@ -16,9 +16,8 @@ export type ProtocolParametersRequiredByWallet = Required<
   >
 >;
 
-export type AssetSupply = {
+export type SupplySummary = {
   circulating: Cardano.Lovelace;
-  max: Cardano.Lovelace;
```

Comment:
Removed from the interface since it's already exposed by [CompactGenesis](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/core/src/Cardano/types/Genesis.ts#L9-L19)